extension/src/test/e2e/util.ts

```diff
@@ -35,28 +56,42 @@ export const waitForViewContainerToLoad = async () => {
     return !!view
   })
 
-  return browser.waitUntil(async () => {
+  await browser.waitUntil(async () => {
     const numberOfProgressBarsInContainer = 7
-    const currentProgressBars = await $$('.monaco-progress-container')
+    const currentProgressBars = await findProgressBars()
 
-    if (
+    return !(
       currentProgressBars.length <
       initialProgressBars.length + numberOfProgressBarsInContainer
-    ) {
-      return false
-    }
+    )
+  })
+
+  await waitForDvcToFinish()
 
-    for (const progress of currentProgressBars) {
-      if ((await progress.getAttribute('aria-hidden')) !== 'true') {
+  const workbench = await browser.getWorkbench()
+  await workbench.executeCommand('DVC: Pull')
```

Comment:
[F] In the CI we need to pull all of the DVC tracked data. This was a headache to get working.